.github/workflows/unicode-check.yml

```diff
@@ -0,0 +1,22 @@
+name: Unicode check
+on:
+    pull_request:
+        branches:
+            - JSON
+    push:
+        branches:
+            - JSON
+
+jobs:
+    check-unicode:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+
+            - name: Check for unicode in Sail files
+              run: |
+                if git diff --exit-code --name-only ${{ github.base_ref }}..${{ github.head_ref }} | grep -E '\.sail$' | xargs grep -Pl "[^\x00-\x7F]" --null --files-with-matches; then
```

Comment:
Nice. I'm wondering if this is too big of a big hammer, though, preventing innocuous uses of non-ASCII characters in "don't care" areas like introductory comments that have in the past included names like "Gonzàlez".

Should we instead look at the output of "make json"? (I know that will miss areas of _future_ interest, so also not ideal.)